procedures: add Device Auth Tokens documentation (#3168)
eclipse-che/che-docs: modules/administration-guide/partials/proc_setting-up-the-github-oauth-app.adoc

. Copy and save the *GitHub OAuth Client Secret* for use when applying the GitHub OAuth App Secret.

> . To enable the device authorization flow for the Dashboard, scroll to *Enable Device Flow* and select the checkbox, then click *Update application*.

Reviewer: The other steps in this procedure each cover one action. This step combines scroll, select a checkbox, and click a button. Consider splitting:

```suggestion
. Scroll to the *Device flow* section and select *Enable Device Flow*.

. Click *Update application*.
```

Also: an admin who completed the initial OAuth App setup previously will not revisit this procedure page to find the new step appended at the end. Would it be possible to mention Device Flow in the introductory text of `configuring-oauth-2-for-github.adoc`, or add an admonition there, to surface it to admins who already completed the base setup?

. Copy and save the *GitHub OAuth Client Secret* for use when applying the GitHub OAuth App Secret.

> . Scroll to the *Device flow* section and select the *Enable Device Flow* checkbox.

Reviewer: The two new numbered steps are presented as regular procedure steps in a sequential setup flow, which implies they are mandatory for all admins. The NOTE says "This step is required **if** you want users to connect their GitHub accounts..." - the conditional "if" makes it optional.

An admin following the steps top-to-bottom will enable Device Flow regardless of intent, or conversely might skip it because the NOTE makes it sound non-essential. Consider either marking the steps `*(Optional)*` or removing the conditional language from the NOTE if Device Flow is always recommended.

+

> NOTE: This step is required if you want users to connect their GitHub accounts from the {prod-short} Dashboard using *User Preferences > Device Auth Tokens*.

Reviewer: The NOTE tells admins that Device Flow enables users to connect from *User Preferences > Device Auth Tokens* but does not link to the user-facing documentation for that feature. Consider adding an xref to `end-user-guide:connecting-to-github-using-device-authorization.adoc` so an admin can navigate directly from the admin guide to the feature they just enabled.
